packages/driver/src/multi-domain/events/misc.ts

```diff
@@ -11,7 +11,10 @@ export const handleMiscEvents = (Cypress: Cypress.Cypress, cy: $Cy, specBridgeCo
   })
 
   specBridgeCommunicator.on('viewport:changed:end', () => {
-    if (viewportChangedCallbackFn) viewportChangedCallbackFn()
+    if (viewportChangedCallbackFn) {
+      viewportChangedCallbackFn()
+      viewportChangedCallbackFn = null
```

Comment:
Do we need to reset this after a single use, or after the domain context is finished? Or does the callback always get passed back in anytime `viewport:changed` is emitted making this a non issue?

Reply:
The callback get passed in for every `viewport:changed` event.

Reply:
OK sweet guess it's a non issue!